targets/csharp/source/source/Extensions/Models/PlayFabCloudScriptModelsExtensions.cs.ejs

```diff
@@ -0,0 +1,57 @@
+∩╗┐namespace PlayFab.CloudScriptModels
+{
+    public class FunctionExecutionContext<T>
+    {
+        public PlayFabApiSettings ApiSettings { get; private set; }
+
+        public ProfilesModels.EntityProfileBody CallerEntityProfile { get; set; }
+
+        public T FunctionArgument { get; set; }
+
+        protected FunctionExecutionContext() { }
+
+        public static async System.Threading.Tasks.Task<FunctionExecutionContext<T>> Create(System.Net.Http.HttpRequestMessage request)
+        {
+            string body = await request.Content.ReadAsStringAsync();
+            var contextInternal = Json.PlayFabSimpleJson.DeserializeObject<FunctionExecutionContextInternal<T>>(body);
+            var settings = new PlayFabApiSettings
+            {
+                TitleId = contextInternal.TitleAuthenticationContext.Id,
+                DeveloperSecretKey = contextInternal.TitleAuthenticationContext.SecretKey
+            };
+
+            // TODO: STATIC ENTITY TOKEN?
```

Comment:
We don't want to check this in, right?

Reply:
Switched to `PlayFabAuthenticationContext` field as per our conversation.